### DISCUSSION TITLE
New features coming in the next update

### DISCUSSION THREAD

**Discussion by moudey** [moudey/Shell#30] [News]
![20220827_075004](https://user-images.githubusercontent.com/39198/187016898-df6499f6-52ba-495f-a3fe-86fa359a069a.jpg)
![20220827_075011](https://user-images.githubusercontent.com/39198/187016900-59f32329-ee23-4969-bc99-a7f62c4b2994.jpg)
![20220827_075007](https://user-images.githubusercontent.com/39198/187016902-98ce2aa9-90e7-44db-8974-e4967fd8c968.jpg)

**Comment by HuyHung1408** [comment]
cool, good job!!

**Reply by CyCoSyS** [reply]
I'm really enjoying your Shell app.  Getting my Context menu just right.  Wasted 9 hours on a snowy storm day, and had a wonderful time putzing around and experimenting.  Great work!

**Reply by moudey** [reply]
> I'm really enjoying your Shell app. Getting my Context menu just right. Wasted 9 hours on a snowy storm day, and had a wonderful time putzing around and experimenting. Great work!

Thank you. Stay tuned for the next update as it will introduce many new features and improvements.

**Comment by Jai-JAP** [comment]
Does this also include mica effect?

**Comment by moudey** [comment]
Hi all, Acrylic and Blur effects added

Acrylic effect
![acrylic](https://user-images.githubusercontent.com/39198/187549954-c7ed3a71-db85-4da3-94ab-8b67b5557643.png)

Blur effect
![blur](https://user-images.githubusercontent.com/39198/187549959-5e2c6e61-9ce2-4680-9e7f-59ed2a4c1445.png)
